instance/tests/integration/integration_instance.py

```diff
@@ -107,6 +110,29 @@ def test_external_databases(self):
         self.assertTrue(instance.successfully_provisioned)
         self.assertFalse(instance.require_user_creation_success())
 
+    @shard(3)
```

Comment:
Currently we only have shards 0, 1 and 2 on CircleCI.  tests configured to run on shard 3 will run on shard 0 instead.  You could add another container on CircleCI (I think we can get 4 instead of three, but I'm not sure).